htlcswitch/link.go

```diff
@@ -1898,7 +1898,9 @@ func (l *channelLink) updateCommitTx() error {
 		CommitSig: theirCommitSig,
 		HtlcSigs:  htlcSigs,
 	}
-	l.cfg.Peer.SendMessage(false, commitSig)
+	if err := l.cfg.Peer.SendMessage(false, commitSig); err != nil {
```

Comment:
Yes, I am aware of that (see commit message). Just added it for completeness in case the implementation may change in the future. It is an unimportant commit.